docs/src/cookbooks/cookbook.md

```diff
@@ -507,7 +478,7 @@ class CustomBundleFixed(elts: (String, Data)*) extends Record {
 }
 ```
 
-### How do I create a finite state machine (FSM)?
+## How do I create a finite state machine (FSM)?
```

Comment:
There were accidentally getting grouped under the `Bundles` heading. See screenshot below of the current version of this page (ie. the one live on the website right now). See how we manually didn't group them under Bundles, but the autogenerated sidebar does have them under it.

<img width="927" alt="Screenshot 2024-01-29 at 9 25 45 PM" src="https://github.com/chipsalliance/chisel/assets/8584418/7fd024fe-4c58-4dd2-a843-f072eecf991c">